spec/models/binary_blob_part_spec.rb

```diff
@@ -21,4 +21,14 @@
       expect(subject.bytes.to_a).to eq(@data.bytes.to_a)
     end
   end
+
+  it "#inspect has data filtered out" do
+    str = "REALLY LONG STRING"
+    part = FactoryBot.create(:binary_blob_part, :data => str)
+    expect(part.inspect).to_not include(str)
+
+    # Also check for how str is stored in data_before_type_cast, in case that
+    # leaks through to inspect
+    expect(part.inspect).to_not include("5245414c4c59204c4f4e4720535452494e47")
```

Comment:
what do these numbers mean?

Reply:
> how str is stored in data_before_type_cast

This is the raw value stored in the database.

Reply:
is there any way to codify that - as in transform `str` into `"5245414c4c59204c4f4e4720535452494e47"`?

Reply:
I just updated the OP to reflect the same number so it's a little clearer.  If you look there for 5245... you will see what I mean.  Had we had these tests in place originally, then this would have failed.

Reply:
Also updated the comment

Reply:
> is there any way to codify that - as in transform str into "5245414c4c59204c4f4e4720535452494e47"

That's a good idea...I'll try.

Reply:
Done.  Thanks!

Reply:
```ruby
[1] pry(main)> s = "REALLY LONG STRING"
[2] pry(main)> s.unpack1("H*") == "5245414c4c59204c4f4e4720535452494e47"
=> true
```